### DISCUSSION TITLE
太记是否应该建立一个官网？

### DISCUSSION THREAD

**Discussion by fcjz** [tiddly-gittly/TidGi-Desktop#294] [Ideas]
notion、obsidian、思源笔记等知识管理工具都有自己的官方网站，我想官方网站可以增加太记的知名度，让更多人认识到太记。

目前太记的下载方式除了爱好者qq群之外就是GitHub，大多数知识管理爱好者接触不到太微（虽然目前国内对太微的宣传增多了，但和其它工具比起来还是十分冷门的，我看知乎有一个问题说的是太记、思源等几个笔记工具的比较，答主几乎都围绕思源等进行讨论，都没有看见过太记……），也大概率不会因为一时的接触便加入qq群；大多数知识管理爱好者不是开发者，不熟悉GitHub，可能不很会操作。

我个人认为建设官网是有必要的（哪怕官网是简陋的），起码我自己，在搜索一个工具时，也倾向于点击官网而不是GitHub页面。

我在discussion和issue搜索了“官网”相关的关键词，没有结果，所以我来开启了这样一个讨论，我不知道开发者对此有什么意见，欢迎说说你的看法。

**Comment by linonetwo** [comment]
我没啥空搞，社区有感兴趣的可以搞一个，可以看到这个组织里已经有不少别的小伙伴写的项目了。

而且目前还是以吸引有共建能力的开发者为主，普通用户随缘了。以后自动答疑机器人有人写了，还有所见即所得等插件再更完善了，再考虑吸引更多竞品的用户。

**Reply by linonetwo** [reply]
最近功能比较完善了，所以还是来搞一个 https://github.com/tiddly-gittly/TidGi-Desktop/issues/348

**Reply by fcjz** [reply]
可以可以，所以你是要弄成群里发的TW预备官网示例那样？

**Reply by linonetwo** [reply]
是的

**Reply by linonetwo** [reply]
再用 oeyoews 分享的 通过URL切换布局 https://neotw.oeyoewl.top/?gettingstarted
